en/cobalt/_posts/user/2015-02-15-template-tips-and-tricks

```diff
@@ -0,0 +1,115 @@
+---
+layout: doc
+title:  "Templates tips and tricks"
+date:   2015-02-15 09:30:00
+tags: templates
+intro: "Additional information."
+---
+
+# Template tips & tricks
+
+Here are some examples on how to modify Cobalt templates.
+
+## Markup template
+
+You can find markup template in *components/com_cobalt/views/records/tmpl/default_markup_default.php*
+
+#### Remove dropdown Switch view and change template labels to images with labels as tooltips
+
+So let's try do change this template from
+
+![Dropdown Switch view markup template](http://demo.web-er.si/markup_template_template_switcher_before.jpg)
```

Comment:
you use image here and also later. Please store screenshot in `assets/img/screenshots` and then use something like this

    ![Dropdown Switch view markup template](/assets/img/screenshots/markup_template_template_switcher_before.jpg)

This way we ensure screenshot availability.